proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -112,3 +128,45 @@ message QueryValidatorProviderAddrResponse {
   // The address of the validator on the provider chain
   string provider_address = 1;
 }
+
+message QueryPendingSlashPacketsRequest {}
+
+message QueryPendingSlashPacketsResponse {
+  // current slash_meter state
+  int64 slash_meter = 1;
+  // allowance of voting power units (int) that the slash meter is given per replenish period
+  // this also serves as the max value for the meter.
+  int64 slash_meter_allowance = 2;
+  google.protobuf.Timestamp last_replenish = 3
+    [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+  google.protobuf.Timestamp next_replenish = 4
```

Comment:
See comment below about possibly removing this field in favor of last full time and the replenishment period